pkg/proxy/winkernel/proxier.go

```diff
@@ -1010,13 +1038,16 @@ func (proxier *Proxier) syncProxyRules() {
 		containsNodeIP := false
 
 		for _, epInfo := range proxier.endpointsMap[svcName] {
-
 			ep, ok := epInfo.(*endpointsInfo)
 			if !ok {
 				klog.Errorf("Failed to cast endpointsInfo %q", svcName.String())
 				continue
 			}
 
+			if !ep.IsReady() {
+				continue
+			}
```

Comment:
@kumarvin123 @ksubrmnn This would be really cool to support in Windows as well in an upcoming release.

Reply:
FYI @jayunit100

Reply:
Looks like there's an issue for windows support here https://github.com/kubernetes/kubernetes/issues/96514, thanks @jayunit100